pom.xml

```diff
@@ -339,6 +339,7 @@
 				<module>spring-cloud-gcp-vision</module>
 				<module>spring-cloud-gcp-kms</module>
 				<module>spring-cloud-gcp-native-support</module>
+				<module>spring-cloud-spanner-spring-data-r2dbc</module>
```

Comment:
check if we want to add it in the release module

Reply:
`spring-cloud-spanner-spring-data-r2dbc` is currently not in the release profile
@zhumin8 @meltsufin Shall we enable the release and disable this module's release from `[cloud-spanner-r2dbc](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc/tree/main/cloud-spanner-r2dbc)` ?

Reply:
Yes, this is targeting 3.x branch, I think we can add it to release profile as part of this pr. And after successfully released, remove it form the old repo release profile.
Note that `cloud-spanner-r2dbc` repo does not have a scheduled releases, we can technically remove it from that release profile at any time.

Reply:
sure @zhumin8 , I have added `spring-cloud-spanner-spring-data-r2dbc` to release profile in this PR itself. CC @meltsufin